src/lib/analyzeOas.ts

```diff
@@ -130,12 +136,34 @@ const OPENAPI_FEATURE_DOCS: Record<keyof Analysis['openapi'], Pick<AnalyzedFeatu
     },
   },
   xml: {
-    description: 'Any parameter and/or request body that accepts XML or responses that return XML payloads.',
+    description:
+      'Any schema that utilizes the XML object for defining its shape in an XML payload, or a request or response that defines an XML payload.\n\nThis is deprecated in favor of `xmlSchemas`, `xmlRequests`, and `xmlResponses`.',
+    url: {
+      '3.0': 'https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.4.md#xml-object',
+      '3.1': 'https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.1.md#xml-object',
+    },
+  },
+  xmlSchemas: {
+    description: 'Any schema that utilizes the XML object for defining its shape in an XML payload.',
     url: {
       '3.0': 'https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.4.md#xml-object',
       '3.1': 'https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.1.md#xml-object',
     },
   },
+  xmlRequests: {
+    description: 'Any request body that accepts an XML payload.',
+    url: {
+      '3.0': 'https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.4.md#media-type-object',
+      '3.1': 'https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.1.md#media-type-object',
```

Comment:
same thing down here:

```suggestion
      '3.0': 'https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.4.md#request-body-object',
      '3.1': 'https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.1.md#request-body-object',
```